pkg/routeversion/version.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Nit: s/2017/2022

Reply:
it is a copy of already existing class, but I guess we can put 2022 since this is a common generic code